control/utils.py

```diff
@@ -182,6 +183,24 @@ def is_valid_nqn(nqn):
 
         return (0, os.strerror(0))
 
+class GatewayUtilsCrypto:
+    ENCRYPTION_KEY = b"Q\xba\x7f._i\x9f%\xed\x96\xe0\xaf\xab k\x9e\xd6\xb6c\xb8\xee\xf6\xa7\xb1s\xf7\xf4\x89f'\x85b"
+    secret_box = None
```

Comment:
Even admitting to somehow store the encryption key together with the source code, the class interface should already be open to allow custom keys:
```suggestion
class GatewayUtilsCrypto:
    def __init__(self, encryption_key: bytes):
        # we shouldn't store the encryption key if the security library
        # already provides a mechanism to keep it
        self.__secret_box = nacl.secret.SecretBox(encryption_key)
```